test/js/node/http/node-http.test.ts

```diff
@@ -2435,3 +2435,118 @@ it("should work when sending https.request with agent:false", async () => {
   await promise;
 });
 
+it("client should use chunked encoded if more than one write is called", async () => {
+  using server = Bun.serve({
```

Comment:
Can you write this test in a way that runs in Node too? This file has had many tests which did not actually run in Node

Reply:
You mean using node:http server instead of Bun?

Reply:
done